### ISSUE TITLE
Remove depreciated addons

### ISSUE THREAD

**Issue by AndyHee** [friendica/friendica#4427]
There are several addons that appear depreciated or beyond any conceivable usage.

**Comment by AndyHee** [comment]
Michael already flagged this one up: https://github.com/friendica/friendica-addons/pull/523

**Comment by annando** [comment]
Would be great if you can do a little check and list them up here.

**Comment by AndyHee** [comment]
Of the top of my head.

**Infinite Improbability Drive** 
Never worked with Frio, intermittently worked with Vier 

**Mahjongg**
Never worked with Frio or Vier

**Comment by AndyHee** [comment]
**Blockem**

I  might not fully understand this in which case the document may need some attention.  Or possibly its function where incorporated into the core without removing the addon.

This addon works at user level. Users can block people via adding a profile URL. But blocking people is normally done via the contact settings.

**Comment by AlfredSK** [comment]
> This addon works at user level. Users can block people via adding a profile URL. But blocking people is normally done via the contact settings.

Blockem and Superblock are working differently. The benefit of these two is **you don't have to add someone as a contact** to completely block the user from your timeline.
Think of a troll who is commenting a lot of public posts. Add his profile to one of these addons and you'll never see his posts or comments.

That being said, both addons do not work reliable anymore. I see a lot of comment from superblocked profiles lately.

**Comment by MrPetovan** [comment]
AFAIK blockem just hides the posts' content under a nsfw-like accordion.

**Comment by tobiasd** [comment]
> That being said, both addons do not work reliable anymore. I see a lot of comment from superblocked profiles lately.

That is really sad! Their functionality should be fixed for the next release.

And regarding Mahjongg, do you have flash activated? As it is a flash game that works just fine with vier.

**Comment by AndyHee** [comment]
> Blockem and Superblock are working differently.

OK, but what is superblock?

We have two admin tools 1. Contact Blocklist 2. Server blocklist.

Plus two user tools: 1. Blockem (hiding post via accordion) whoever 2. Block contact (blocking your own contacts) 

All seem useful, but may need some renaming and better documentation. I also wonder if this hiding function should not just be integrated in the core function and readily accessible via the menu in contact.

**Comment by AndyHee** [comment]
> AFAIK blockem just hides the posts' content under a nsfw-like accordion.

Yes, it instantaneously hides the _content_ of posts, including those already in your timelines.

Here a test: https://pasteboard.co/H6Omi0V.png It definitely seems odd to call this hiding of content "blocked" while the post itself is still there and while we have other functions called blocking too.

(Test over! Steffen's url removed.)

**Comment by AlfredSK** [comment]
Where's my avatar? 😱

**Comment by tobiasd** [comment]
> OK, but what is superblock?

blockem collapses the postings and superblock does not show them anymore at all, AFAIR

**Comment by tobiasd** [comment]
> All seem useful, but may need some renaming and better documentation.
> I also wonder if this hiding function should not just be integrated
> in the core function and readily accessible via the menu in contact.

The addons work for non-contacts. While the ignore function in the contacts context only works for those you are connected with. So if you have someone spamming a forum, or the postings of a OStatus contact, you can block them with the addons.

**Comment by AndyHee** [comment]
Now I understand. Superblock another addon. This blocks anyone regardless if they are contacts. Right?

**Comment by AndyHee** [comment]
> Where's my avatar? 😱

Yes that's part of its function. "Route 66" now clearly visible again.

**Comment by AndyHee** [comment]
I've created a PR https://github.com/friendica/friendica-addons/pull/527 that's trying to improve Blockem's wording and description.

**Comment by AndyHee** [comment]
I'm looking at this addon: _forumdirectory_.

It's activated on my node, but it there seems to be no way of actually configuring or using it. Does anyone know more about this? Is it deprecated or merely lacks sufficient documentation?

**Comment by tobiasd** [comment]
Just have a look at https://friendica.hubup.pro/forumdirectory as logged in user you should see a directory of local forums there.

**Comment by tobiasd** [comment]
Also, frio theme seems to not provide the "Apps" menu, thus hiding addons that hook into that menu.

**Comment by AndyHee** [comment]
Thanks Tobias!

I gather there is no configuration and the displayed information about each forum is taken from the 
"About" field in each profile.

I'll expand this line a bit to clarify this. Plus add a note that in Frio the functionally is currently limited.

> Add a directory of forums hosted on your server, with verbose descriptions.

**Comment by AndyHee** [comment]
I understand the Superblock addon is now redundant.

Shall I remove it? 

If so, I will open pull requests to move it here: https://github.com/friendica/deprecated-addons and delete here:  https://github.com/friendica/friendica-addons

**Comment by MrPetovan** [comment]
Yes please.

**Comment by AndyHee** [comment]
It looks like https://github.com/friendica/deprecated-addons  is now an archive:

`
This repository has been archived by the owner before Nov 9, 2022. It is now read-only. `

What shall we do?

**Comment by MrPetovan** [comment]
Please submit deprecated addons to Gitea: https://git.friendi.ca/friendica/deprecated-addons

**Comment by AndyHee** [comment]
Done: https://git.friendi.ca/friendica/deprecated-addons/pulls/4

**Comment by AndyHee** [comment]
Blockem now removed https://git.friendi.ca/friendica/friendica-addons/pulls/1391
and preserved here: https://git.friendi.ca/friendica/deprecated-addons/pulls/5

Relates to https://github.com/friendica/friendica-addons/pull/1347

**Comment by AndyHee** [comment]
I have identified some more addons that seems deprecated. Any thoughts on this?

### Smileybutton
https://github.com/friendica/friendica-addons/tree/develop/smileybutton

Since we have a very neat emoji function in the core, `Smileybutton` seems redundant. Also, these smilies seem from a long bygone era and the implementation of this addon is very crude in terms of its UX.

### smiley_pack and smilies_adult

Here, I'm not entirely sure whether these two addons are free-standing addons or just extensions of `Smileybutton`

https://github.com/friendica/friendica-addons/tree/develop/smilies_adult
https://github.com/friendica/friendica-addons/tree/develop/smiley_pack
